src/types.ts

```diff
@@ -51,12 +51,17 @@ export type EligibilityPlanToDisplay = EligibilityPlan & {
 
 export enum Locale {
   en = 'en',
+  'fr-FR' = 'fr-FR',
```

Comment:
I think this is not the Widget job to check which locale is allowed or not,
We could inform that waited format is `locale-COUNTRY` or `locale` only (`fr-FR` or `fr` only) ... but we should not refuse an un-waited locale here (just fallback on default `en` if not found)

Reply:
This is just for the typing, you can put fr-BE and it'd be displayed in fr.

Reply:
and if you put : 'idontknow' it would be in english also